New package: SoftMaker.FreePDF version 2025.407

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #263800 

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

<details><summary>Screenshots</summary><p>

![Image](https://github.com/user-attachments/assets/67b32ff5-94d2-4c3c-97a1-044df13483eb)
![Image](https://github.com/user-attachments/assets/f609442f-5656-431c-83ea-cd5d118274fb)


</p></details>

-----


 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/263804)

Dragon1573 committed:
New package: SoftMaker.FreePDF version 2025.407

Dragon1573 committed:
Update manifest: SoftMaker.FreePDF version 2025.407

- Correction of package identifier

Dragon1573 commented:
> Template: msftbot/validationError/installation/unattended

- See https://github.com/microsoft/winget-pkgs/pull/263804#issue-3127069056:~:text=Manual%20validation

😕 All validations works as expected.

stephengillie commented on manifests/s/SoftMaker/FreePDF/2025.407/SoftMaker.FreePDF.installer.yaml:
```diff
@@ -0,0 +1,19 @@
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\SoftMaker\FreePDF 2025'
+Installers:
+- Architecture: x86
```
```suggestion
- Architecture: x64
```

stephengillie reviewed (changes requested):
> Automatic Validation ended with:
> 
> > 2025-06-07 08:04:58.690   Exception message:                           Internal error: Expression error 'Runtime error (at 86:272):                           Internal error: Cannot access 64-bit registry keys on this version of Windows.'
> 
> (Automated response - build 1064.)

Dragon1573 committed:
Apply suggestions from code review

- Switch architecture label: `x86` -> `x64`
  - The installer can't touch `x64` Windows Registry Key in `x86` VM instance
  - The application MUST be `x64` arch

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>
